taccsite_ui/fractal.config.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+const path = require('path');
+const mandelbrot = require('@frctl/mandelbrot');
+const minimist = require('minimist');
+
+// Get base config
+const fractal = require('@tacc/core-styles/fractal.config.js');
+const themeConfig = require('./fractal.theme.js');
+
+// Get project
+// (name)
+const args = minimist( process.argv.slice( 2 ));
+let cmsName = 'core-cms';
+let projName = args['project'] || '';
+    projName = ( projName !== cmsName ) ? projName : '';
+// (stylesheet)
+const cmsCSSFile = 'cms/site.css';
+const projCSSFile = ( projName )
+  ? `projects/${projName}/static/${projName}/css/build/site.css`
+  : null;
+
+// Set source paths
+// (for components)
+fractal.components.set('exclude', '*.md');
+fractal.components.set('path', __dirname + '/patterns');
```

Comment:
This is directory that just has symlinks. Fractal can not read from more than one directory.

[known issue](https://github.com/frctl/fractal/issues/207)